netbox/dcim/forms/filtersets.py

```diff
@@ -977,9 +977,10 @@ class CableFilterForm(TenancyFilterForm, NetBoxModelFilterSetForm):
         label=_('Color'),
         required=False
     )
-    length = forms.IntegerField(
+    length = forms.DecimalField(
         label=_('Length'),
-        required=False
+        required=False,
+        min_value=0.01
```

Comment:
Per #13649 we allow cable lengths to be set to zero, so the filter should support this as well.
```suggestion
        required=False
```